app/models/publication.rb

```diff
@@ -159,13 +140,9 @@ def build_from_sciencewire_hash(new_sw_pub_hash)
     self
   end
 
-  def build_from_pubmed_hash(new_pubmed_pub_hash)
-    self.pub_hash = new_pubmed_pub_hash
-    self
-  end
```

Comment:
Why is this entirely removed?  What replaces this?  How do you know it's not called anywhere?

Reply:
Read the commit message.  I know it's not called because I located the couple places that called it.  Because the pubmed case doesn't require any special logic, it doesn't need an extra method to accomplish what can be done with just basic initialization or assignment, like:
```ruby
obj = Publication.new(pub_hash: new_hash1)
obj.pub_hash = new_hash2
```